pymc_marketing/clv/utils.py

```diff
@@ -0,0 +1,118 @@
+from typing import Union
+
+import numpy as np
+import pandas as pd
+import xarray
+
+
+def to_xarray(customer_id, *arrays, dim: str = "customer_id"):
```

Comment:
I'm curious why this is no longer an internal method for the `CLVModel` base class?

Reply:
It was an internal method of the `BetaGeoModel` only

Reply:
My mistake. Do you think this would be more appropriate as an internal method?